packaging/Makefile_distro_vars.mk

```diff
@@ -91,6 +91,14 @@ DISTRO_VERSION  ?= $(VERSION_ID)
 ORIG_TARGET_VER := 15.5
 SED_EXPR        := 1p
 endif
+ifeq ($(CHROOT_NAME),opensuse-leap-15.6-x86_64)
+VERSION_ID      := 15.6
+DISTRO_ID       := sl15.6
+DISTRO_BASE     := LEAP_15
+DISTRO_VERSION  ?= $(VERSION_ID)
+ORIG_TARGET_VER := 15.6
+SED_EXPR        := 1p
+endif
```

Comment:
is this an import from daos-stack/packaging/ or an addition on top of it? just want to make sure that next update of packaging wont eliminate this

Reply:
It is an import from daos_stack/packaging.